src/perception/RcnnMopedModule.cpp

```diff
@@ -0,0 +1,163 @@
+#include <aikido/perception/RcnnMopedModule.hpp>
+
+#include <Eigen/Geometry>
+#include <dart/utils/urdf/DartLoader.hpp>
+#include <ros/ros.h>
+#include <ros/topic.h>
+#include <visualization_msgs/Marker.h>
+#include <visualization_msgs/MarkerArray.h>
+#include <aikido/perception/shape_conversions.hpp>
+#include <aikido/io/CatkinResourceRetriever.hpp>
+
+namespace aikido {
+namespace perception {
+
+RcnnMopedModule::RcnnMopedModule(
+    ros::NodeHandle nodeHandle,
+    std::string markerTopic,
+    dart::common::ResourceRetrieverPtr resourceRetriever,
+    std::string canUri,
+    std::string referenceFrameId,
+    dart::dynamics::Frame* referenceLink)
+  : mNodeHandle(std::move(nodeHandle))
+  , mMarkerTopic(std::move(markerTopic))
+  , mReferenceFrameId(std::move(referenceFrameId))
+  , mResourceRetriever(std::move(resourceRetriever))
+  , mCanUri(std::move(canUri))
+  , mReferenceLink(std::move(referenceLink))
+  , mTfListener(mNodeHandle)
+{
+  // Do nothing
+}
+
+
+//=============================================================================
+const dart::dynamics::SkeletonPtr makeBodyFromURDF(const std::string& uri,
```

Comment:
We have this helper function in a few places already, so perhaps we should create `aikido::io::util` and add this method there. When we do that, we should add a `resourceRetriever` argument (so we don't have the overhead of initializing the `CatkinResourceRetriever` multiple times) like here: https://github.com/personalrobotics/herb3_demos/blob/master/soda_handoff/src/soda_handoff.cpp#L36

Reply:
Removed this part. I used dart::utils::DartLoader instead.